content/en/agent/guide/how_remote_config_works.md

```diff
@@ -150,23 +134,71 @@ DD_API_KEY=xxx
 DD_REMOTE_CONFIGURATION_ENABLED=true
 ```
 
-To enable Datadog Remote Instrumentation, add the following to your Helm Chart.
+{{% /tab %}}
+{{% tab "Helm" %}}
+Add the following to your Helm chart, specifying the API key that has Remote Config capability enabled:
 ```yaml
-clusterAgent:
-  admissionController:
-    remoteInstrumentation:
-      enabled: true
+datadog:
+  apiKey:  # <DATADOG_API_KEY>
+  remoteConfiguration:
+    enabled: true
 ```
+
 {{% /tab %}}
 {{< /tabs >}}
 
+
+6. Restart your Agent for the changes to take effect. 
+
 After you perform these steps, your Agent requests its configuration from Datadog, and the features that use remote configuration are enabled:
 - [CWS default agent rules][5] update automatically as released.
-- Datadog Remote Instrumentation is enabled.
+- [Datadog Remote Instrumentation][13] is enabled.
 - [APM Agent-level sampling rates][6] are applied.  
 - [Dynamic Instrumentation][7] is enabled.
 - [ASM 1-Click enablement, IP blocking, and attack pattern updates][8] are enabled.
 
+## Best practices
+
+### Datadog Audit Trail
+
+Use [Datadog Audit Trail][14] to monitor organization access and Remote Configuration enabled events. Audit Trail allows your administrators and security teams to track the creation, deletion, and modification of Datadog API and application keys. After Audit Trail is configured, you can view events related to Remote Configuration enabled features and who has requested these changes. Audit Trail allows you to reconstruct sequences of events, and establish robust Datadog monitoring for Remote Configuration. 
+### Monitors
+Configure [monitors][15] to receive notifications when an event of interest is encountered.
+
+## Troubleshooting
+
+### Restart the Agent
+
+After the Agent configuration is updated in the [`datadog.yaml`][16] file, restart the Agent for this change to take effect. 
+
+### Ensure Datadog Config endpoints are reachable from your environment	
+
+To perform Remote Configuration, both the Agent and  the Observability Pipelines Worker deployed in your environment communicate to Datadog Config [endpoints][17]. Ensure that outbound HTTPS has access to these endpoints from your environment. If you also have a proxy in between Datadog and your environment, update your proxy settings to incorporate Config destinations.
```

Comment:
Hey @estherk15 there is an extra space here and I am not sure how to get past this error, if I pull the changes locally, its out of date and won't let me update it because it shows my branch is behind. How can I just remove the extra space and get this check to pass ?